anta/tests/configuration.py

```diff
@@ -75,3 +76,55 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(command_output)
+
+
+class VerifyRunningConfigLines(AntaTest):
+    """Verifies the given regular expression patterns are present in the running-config.
+
+    **Since this uses regular expression searches on the whole running-config, it can
+    drastically impact performance and should only be used if no other tests is available for.**
+    If possible, try using another Anta test that is more specific.
```

Comment:
```suggestion
    !!! warning
    Since this uses regular expression searches on the whole running-config, it can
    drastically impact performance and should only be used if no other tests is available for.
    If possible, try using another ANTA test that is more specific.
```

Reply:
If you do `mkdocs serve` locally you can see the documentation. `!!! warning` in `mkdocs` adds a nice little box.